OWNERS_ALIASES

```diff
@@ -161,6 +161,12 @@ aliases:
     - erickhun
     - jygastaud
     - awkif
+  sig-docs-it-owners: #Team: Italian docs localization; GH: sig-docs-it-owners
```

Comment:
@micheleberardi Please add yourself to these aliases as well. Before this PR can merge, you must first [open an issue](https://github.com/kubernetes/org/issues) to join the Kubernetes GitHub org and be approved to join. I'm happy to sponsor you; @rlenferink 👋 are you willing to co-sponsor?

Reply:
Hi guys how are u? 
I’m traveling in west cost these days I will back in NYC Thursday night 
I can do that when I back 
It’s ok or too late ? 
Thanks

Reply:
I did all steps above 
Thanks for the support

Reply:
Thanks for the review @zacharysarah 
I am willing to co-sponsor @micheleberardi